test/test_gmtime.c

```diff
@@ -0,0 +1,214 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libphoenix
+ *
+ * test/test_strftime.c
+ *
+ * Copyright 2020 Phoenix Systems
+ * Author: Marcin Brzykcy
+ *
+ * This file is part of Phoenix-RTOS&.
+ *
+ * %LICENSE%
+ */
+
+#include "test_common.h"
+#include <time.h>
+#include <stdio.h>
+#include <stdlib.h>
+
+#define NCOLS 9
+#define T1_LEN 50
+
+static const time_t input_vector[T1_LEN];
+static const int output_vector[T1_LEN][NCOLS];
+
+
+/* help function for future test generation if needed - not used in test functions */
+void generate_input_host(void)
+{
+	int i;
+	struct tm t;
+	srand(time(NULL));
+	for (i = 0; i < T1_LEN; i++)
+	{
+		t = (struct tm) { .tm_sec = rand() % 59, .tm_min = rand() % 59, .tm_hour = rand() % 23, 
+						  .tm_mday = rand() % 30, .tm_mon = rand() % 11, .tm_year = 1980 + rand() % 60, 
+						  .tm_wday = 0, .tm_yday = 0, .tm_isdst = 0 };
+		printf("%lld", mktime(&t));
+		if (i != T1_LEN - 1)
+			printf(",\n");
+	}
+	printf("\n");
+}
+
+
+
+/* help function for future test generation if needed - not used in test functions */
+void generate_output_host(void)
+{
+	int i;
+	struct tm *t;
+	printf("Printing host output data. Struct tm member values:\n");
+	for (i = 0; i < T1_LEN; i++)
+	{
```

Comment:
opening bracket should be in the same line. the same rule applies to 'ifs'.
See our [coding convention page ] ( https://phoenix-rtos.com/documentation/codingl) for more info